### PR TITLE
documentation of helm chart usage

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
Documentation reviewed with commit.

File: helm-charts/ovms/README.md [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+By default, Model Server is deployed with 1 instance. If you would like to scale up additional replicas, override the value in values.yaml file or by passing _--set_ flag to _helm install_:
+```shell script
+helm install ovms-app ovms --set models_settings.model_name=resnet50-binary-0001,models_settings.model_path=gs://models-repository,deployment_parameters.replicas=3
+```
+
+### Deploy OpenVINO Model Server with Multiple Models Defined in a Configuration File
+To serve multiple models, you can run Model Server with a configuration file as described in [model server documentation](https://docs.openvino.ai/latest/ovms_docs_multiple_models.html).
+Follow the above documentation to create a configuration file named _config.json_ and fill it with proper information.
+To deploy with config file stored in the Kubernetes ConfigMap:
+* create a ConfigMap resource from this file with a chosen name (here _ovms-config_):
```
[the remark this answers is not in the document]
Comment (dtrawins) [reply helm-charts/ovms/README.md:32]: the link to config.json documentation is already added above

File: helm-charts/ovms/README.md [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+    data = f.read()
+inputs = {"image": data}
+results = client.predict(inputs=inputs, model_name="multiple_vehicle_recognition")
+print("Returned outputs:",results.keys())
+EOL
+```
+Run the prediction via a command:
+```
+$ python pipeline.py
+Returned outputs: dict_keys(['colors', 'vehicle_coordinates', 'types', 'vehicle_images', 'confidence_levels'])
```
[the remark this answers is not in the document]
Comment (dtrawins) [reply helm-charts/ovms/README.md:274]: It was meant to be as simple as possible and is focused on deployment part. Full demo for the pipeline is added as a reference at the beginning.

File: helm-charts/ovms/README.md [hunk 3]
```diff
@@ -86,7 +88,7 @@ You can restrict assigned cluster resources to the OVMS container by setting the
 - `deployment_parameters.resources.requests.memory` - reserved memory allocation
 - `deployment_parameters.resources.requests.xpu_device` - accelerator name like configured in the device plugin - should be the same like set in limits or empty
 - `deployment_parameters.resources.requests.xpu_device_quantity` - number of accelerators - should be the same like set in limits or empty
-By default, there are no restrictions, but that parameter could be used to reduce the CPU and memory allocation. Below is the snippet example from the [values.yaml](https://github.com/openvinotoolkit/model_server/blob/releases/2022/1/deploy/ovms/values.yaml) file:
+Below is the snippet example from the helm chart values.yaml file:
```
Comment (mzegla) [inline helm-charts/ovms/README.md:91]: Why is link to values.yaml removed?

**Review by mzegla — APPROVED** [review]

**Pull request merged**